modules/hydrodyn/src/WAMIT2.f90

```diff
@@ -4221,12 +4221,12 @@ SUBROUTINE Read_DataFile3D( InitInp, Filename3D, Data3D, ErrStat, Errmsg )
                         IF ( EqualRealNos( Data3D%WvDir1(J), Data3D%WvDir2(J) ) .AND. &
                              EqualRealNos( Data3D%WvDir1(K), Data3D%WvDir2(K) ) ) THEN
 
-                              ! Check if not filled
-                           IF ( .NOT. Data3D%DataMask( I, J, K, L )  ) THEN
+                              ! Check if filled
+                              IF ( Data3D%DataMask( I, J, K, L )  ) THEN
```

Comment:
Could you adjust the whitespace on this line?  The `IF` can move over 3 places left

Reply:
Did it work?

Reply:
My apologies, I wasn't specific enough on this.  Line 4225 needed 3 spaces deleted from before the `IF`.  The other lines are good.

Reply:
Sorry, I edited from my phone and did not see the rest of the file. I am gonna fix it properly when I get home.

Reply:
I think it is fixed now!